Dockerfile.dapper203010478

```diff
@@ -0,0 +1,53 @@
+ARG GOLANG=golang:1.16.2-alpine3.12
```

Comment:
Same here, I dont think this file should be checked in

Reply:
Fixed